dev-docs/bidders/greenbids.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: bidder
+title: Greenbids
+description: Prebid Greenbids Bidder Adapter
+biddercode: greenbids
+tcfeu_supported: true
+dsa_supported: true
```

Comment:
Are you sure you pass the ortb DSA params through?

Reply:
const dsa = deepAccess(bidderRequest, 'ortb2.regs.ext.dsa');
    if (dsa) {
      payload.dsa = dsa;
    }